src/index.test.js

```diff
@@ -240,7 +231,8 @@ test('reads text from fs if filePath is provided but not text', () => {
   } catch (e) {
     // ignore
   }
-  expect(fsMock.readFileSync).toHaveBeenCalledTimes(1)
+  // Two hits to .eslintignore
```

Comment:
@kentcdodds It is called one time if I remove `.eslintignore` from this repository or run tests from `src`.. I guess it's an artifact of eslint loading its configuration. It happens just because we run tests from root directory of project. What's the fix you want? a) add to expectations to read eslintignore b) remove .eslintignore c) run tests from src directory and leave `.toHaveBeenCalledTimes(1)`